css/style.css

```diff
@@ -1525,6 +1547,242 @@ input:focus-visible {
   line-height: 1.33;
 }
 
+.single-article-meta {
+  display: grid;
+  grid-template-columns: 1fr 150px 150px;
```

Comment:
@Right-Noe let's think about 'mobile first design' rule here so we do not have to write additional media queries later.